src/rtkpos.c

```diff
@@ -1395,7 +1395,7 @@ static void restamb(rtk_t *rtk, const double *bias, int nb, double *xa)
         if (n<2) continue;
 
         /* in case of ARMODE_WL, use float ambiguity for L1 */
-        if (rtk->opt.modear==ARMODE_WL) {
+        if (rtk->opt.modear==ARMODE_WL||rtk->opt.modear==ARMODE_WL_FIXHOLD) {
```

Comment:
If we do as above suggested, here we don't need to check `rtk->opt.modear==ARMODE_WL_FIXHOLD`.